backend/src/LibExecutionStdLib/StdLib.fs

```diff
@@ -76,11 +76,16 @@ let infixFnMapping : Map<FQFnName.StdlibFnName, (FQFnName.StdlibFnName * Depreca
     ("String", "append", 1), (("", "++"), NotDeprecated)
     ("", "equals", 0), (("", "=="), NotDeprecated)
     ("", "notEquals", 0), (("", "!="), NotDeprecated)
-    // TODO: deprecate when there's a nice way to move from infix fn to language version
-    // ("Bool", "and", 0), (("", "&&"), DeprecatedBecause("Use `&&` instead"))
-    // ("Bool", "or", 0), (("", "||"), DeprecatedBecause("Use `||` instead"))
-    ("Bool", "and", 0), (("", "&&"), NotDeprecated)
-    ("Bool", "or", 0), (("", "||"), NotDeprecated) ]
+    ("Bool", "and", 0),
+    (("", "&&"),
+     DeprecatedBecause(
+       "Use built-in keyword `&&` instead. Use the `convert-to-short-circuiting` command to convert automatically"
```

Comment:
Does the average Dark user know how to 'Use [a] command'? It's never felt very discoverable to me, so I could see a user getting confused by this and not knowing what to do.

I wonder if we could include another field in `DeprecatedBecause` that would allow us to 'link' a Command that would show up when the fn is highlighted. I'm imagining it shows up in the info box, with some link that brings up the command dialog, with that option highlighted.

Maybe overkill?

Reply:
I have intent to followup on this sort of thing, but I think this is the best we can do without increasing the scope of the PR a lot.